thrift/thrift0.13/src/main/java/com/linecorp/armeria/internal/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -188,18 +188,23 @@ ServiceInfo newServiceInfo(Class<?> serviceClass, Iterable<EndpointInfo> endpoin
     }
 
     @Nullable
-    private MethodInfo newMethodInfo(Method method, Iterable<EndpointInfo> endpoints,
+    private MethodInfo newMethodInfo(Method method, Entry entry,
                                      DocServiceFilter filter) {
         final String methodName = method.getName();
-
         final Class<?> serviceClass = method.getDeclaringClass().getDeclaringClass();
         final String serviceName = serviceClass.getName();
         if (!filter.test(name(), serviceName, methodName)) {
             return null;
         }
         final ClassLoader classLoader = serviceClass.getClassLoader();
 
-        final String argsClassName = serviceName + '$' + methodName + "_args";
+        // Need get the function name in thrift proto,
+        // otherwise the argsClassName maybe wrong when use fullcamel compile option
+        String thriftFunctionName = methodName;
+        if (entry.thriftServiceEntry != null) {
+            thriftFunctionName = entry.thriftServiceEntry.functionName(methodName);
+        }
```

Comment:
I think we can probably do:
```suggestion
        final String thriftFunctionName;
        if (entry.thriftServiceEntry != null) {
            thriftFunctionName = entry.thriftServiceEntry.functionName(methodName);
        } else {
            thriftFunctionName = methodName;
        }
```